bundles/AdminBundle/public/css/icons.css

```diff
@@ -239,18 +239,6 @@
     background: url(/bundles/pimcoreadmin/img/flat-white-icons/stop-circle.svg) center center no-repeat !important
 }
 
-.pimcore_nav_icon_php {
-    background: url(/bundles/pimcoreadmin/img/flat-white-icons/php.svg) center center no-repeat !important;
-}
-
-.pimcore_nav_icon_reports {
-    background: url(/bundles/pimcoreadmin/img/flat-white-icons/pie_chart.svg) center center no-repeat !important;
-}
-
```

Comment:
This css class need to be move `.pimcore_icon_systemrequirements`
https://github.com/aashan10/pimcore/blob/fcc9db3687fbd9e7cc6ee4669923f63599a0b1a1/bundles/AdminBundle/public/css/icons.css#L1051-L1053
used in `showSystemRequirementsCheck`

Potentially `pimcore_icon_php` as well